src/test/rustdoc/primitive/no_std.rs

```diff
@@ -0,0 +1,5 @@
+#![no_std]
+
+// @has no_std/fn.foo.html '//a/[@href="{{channel}}/core/primitive.u8.html"]' 'u8'
+// Link to [u8]
```

Comment:
Is this supposed to be an intra-doc link rather than a comment?

Reply:
It was supposed to be a link, good catch!